ui/src/map/layers/utils/switch-layer-utils.ts

```diff
@@ -134,7 +135,7 @@ export function getSwitchRenderer(
             ctx.lineWidth = pixelRatio;
         },
         [
-            (_, [x, y], ctx, { pixelRatio }) => {
+            (_, coord, ctx, { pixelRatio }) => {
```

Comment:
Näitä oli kanssa melko paljon. Sen sijasta että otettaisiin erikseen `x` ja `y` ulos täällä, niin otetaan koko muuttuja ja passataan se `getCoordsUnsafe()`:lle. Logiikkahan ei sinällään muuttunut, mutta on omasta mielestäni nätimpää käsitellä koordinaatteja koordinaatteina, vaikka ne siltikin pitää `getUnsafe()`:ttaa tavalla tai toisella.